eo-maven-plugin/pom.xml

```diff
@@ -253,6 +253,7 @@ SOFTWARE.
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
+          <excludedGroups>Slow</excludedGroups>
```

Comment:
@levBagryansky Could we use lowercase letters only? At least, in the official documentation they use only lowercase: https://junit.org/junit5/docs/current/user-guide/#running-tests-build-maven-filter-tags

Reply:
@volodya-lombrozo agree